web/routes.go

```diff
@@ -63,3 +70,17 @@ func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }
+
+var statuses = [...]string{
+	"In progress",
+	"Done",
+	"Error",
+}
+
+func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
@sandlerben thoughts on doc comments for these handler functions? Realized that I gave one on `initiateTranscriptionJobHandler', even though it's not exported - should we have doc comments for handler functions?